packages/ERTP/src/amountMath.js

```diff
@@ -254,6 +255,7 @@ const AmountMath = {
     assertRemotable(brand, 'brand');
     assertAssetKind(assetKind);
     const value = helpers[assetKind].doMakeEmpty();
+    // @ts-expect-error TS/jsdoc things 'nat' can't be assigned to K subclassing AssetKind
```

Comment:
```suggestion
    // @ts-expect-error TS/jsdoc thinks 'nat' can't be assigned to K subclassing AssetKind
```

Reply:
I thought I fixed this but I see it's not pushed. When you approve I'll do a rebase to fixup the commit this was in and force push. (Holding off on that so I don't make it harder to review)